labels/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"sync"
+
+	"github.com/google/go-github/v28/github"
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/oauth2"
+)
+
+var token = flag.String("token", "", "GitHub access token")
```

Comment:
2/5 we should use an env variable - args tend to get logged.

Reply:
Good suggestion :+1: